ansible.cfg

```diff
@@ -1,2 +1,4 @@
 [ssh_connection]
 pipelining = True
```

Comment:
Is it something that we can/must do to ensure that `ansible` and/or `ansible-playbook` are piking up this ansible.cfg file when they are called by the glue script? Is there an explicit command line argument to do it. If it is so we can use it and drop https://github.com/SUSE/qe-sap-deployment/commit/8a2cd7f25b2fd22dfaa4b8126efc4270ae65174d  at all

Reply:
I'm happy to set this in the glue code and get rid of `ansible.cfg` entirely.

Reply:
The ANSIBLE_PIPELINING is done at this line of PR#67 https://github.com/SUSE/qe-sap-deployment/pull/67/files#diff-bf288212e7712d3f535bcb117d5244f92dfc1318f8a7b8e70be61716743e2273R333